src/module/system/damage/weapon.ts

```diff
@@ -700,18 +700,26 @@ class WeaponDamagePF2e {
             "strike-damage",
             "damage",
         ];
-        if (weapon.isOfType("melee")) return selectors;
+
+        if (weapon.isOfType("melee")) {
+            if (this.strengthBasedDamage(weapon)) {
+                selectors.push("str-damage");
```

Comment:
you're restoring str-based, so should this be str-based, not str-damage?

Reply:
fixed